File: JavaScript/1929. Concatenation of Array.js
/**
 * @param {number[]} nums
 * @return {number[]}
 */
var getConcatenation = function(nums) {
    var n=nums.length;
    var ans=new Array(2*n);
    for(var i=0; i<2*n; i++){
        if(i<n){ans[i]=nums[i];}
        else{ans[i]=nums[i-n];}
    }
    return ans;
};

File: C#/1929. Concatenation of Array.cs
public class Solution {
    public int[] GetConcatenation(int[] nums) {
        int n=nums.Length;
        int[] ans=new int[2*n];
        for(int i=0; i<2*n; i++){
            if(i<n){ans[i]=nums[i];}
            else{ans[i]=nums[i-n];}
        }
        return ans;
    }
}
